tests/test_management.py

```diff
@@ -82,32 +115,28 @@ def test_override_user(self):
             call_command('transmit_course_metadata', '--catalog_user', 'bob')
         assert str(excinfo.value) == error
 
-    @mock.patch(
-        'integrated_channels.integrated_channel.management.commands.transmit_course_metadata.transmit_course_metadata'
-    )
-    def test_working_user(self, mock_data_task):
-        call_command('transmit_course_metadata', '--catalog_user', 'C-3PO')
-        mock_data_task.delay.assert_called_once_with('C-3PO', 'SAP', 1)
-
     @responses.activate
     @freeze_time(NOW)
     @mock.patch('enterprise.api_client.lms.JwtBuilder', mock.Mock())
+    @mock.patch('integrated_channels.degreed.client.DegreedAPIClient.create_course_content')
     @mock.patch('integrated_channels.sap_success_factors.client.SAPSuccessFactorsAPIClient.get_oauth_access_token')
     @mock.patch('integrated_channels.sap_success_factors.client.SAPSuccessFactorsAPIClient.create_course_content')
     def test_transmit_course_metadata_task_with_error(
             self,
-            send_course_import_mock,
-            get_oauth_access_token_mock,
-    ):
+            sapsf_create_course_content_mock,
+            sapsf_get_oauth_access_token_mock,
```

Comment:
@UmanShahzad How come it's not necessary to mock `DegreedAPIClient._get_oauth_access_token` here?

Reply:
Because the OAuth calls are made through the interface methods, i.e. `create_course_content`, so mocking them is sufficient. Not necessarily the case with SAPSF :p

Reply:
@UmanShahzad Ahhh... right :P